### PR TITLE
widget template added in components folder

### PR DESCRIPTION
**Pull request by AdamGrimleyKainos**

### CODE REVIEW

File: client/src/app/components/template/template.component.css [hunk 1]
```diff
@@ -0,0 +1,72 @@
+
+.panel-default{
+    border-top-width: 5px;
+    border-top-color: #027FBB;
+    min-height: 400px;
+    margin-top: 10px;
+
```
Comment (siona) [inline client/src/app/components/template/template.component.css]: Remove extra whitespace please :)

File: client/src/app/components/template/template.component.css [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    border: none;
+    border-radius: 1px;
+    width: 100%;
+    text-align: left;
+    outline: none;
+    overflow: hidden;
+    margin: 0;
+}
+
+
```
Comment (siona) [inline client/src/app/components/template/template.component.css]: One return between each class please - for compactness and consistency

File: client/src/app/components/template/template.component.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+                                <span class="widget-body-txt">detail</span><br>
+                            </button>
+                        </td>
+                    </tr>
+                  </tbody>
+              </table>
+          </div>
+      </div>
+  </div>
+</body>
```
Comment (siona) [inline client/src/app/components/template/template.component.html]: Looks good 👍

File: client/src/app/components/template/template.component.spec.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
```
Comment (siona) [inline client/src/app/components/template/template.component.spec.ts]: I would remove this to have all imports grouped in a block - just being picky :)

File: client/src/app/review-medication/review-medication.component.ts [hunk 5]
```diff
@@ -8,7 +8,6 @@ import { MedicationComment } from '../MedicationComment';
 import { MedicationDescription } from '../MedicationDescription';
 import { Sanitizer } from '@angular/core';
 import { SecurityContext } from '@angular/core';
-import { LoginComponent } from '../login/login.component';
```
Comment (siona) [inline client/src/app/review-medication/review-medication.component.ts]: Is this related to the widget template task or is this just tidying up?

Comment (AdamGrimleyKainos) [reply client/src/app/review-medication/review-medication.component.ts]: I think that was just a change i had that slipped into the commit. Just tidy up of the component

Comment (siona) [reply client/src/app/review-medication/review-medication.component.ts]: No worries - will disregard for now ;)

**Pull request merged**